src/WindowTitle.ps1

```diff
@@ -58,3 +58,45 @@ function Set-WindowTitle {
         }
     }
 }
+
+function Set-TabTitle {
+    [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSUseShouldProcessForStateChangingFunctions", "")]
+    param($GitStatus)
+    $settings = $global:GitPromptSettings
+
+    if ($settings.TabTitle -eq $false) {
+        return
+    }
+
+    # If the user is running Powershell ISE then name the tab
+    if($psISE -and $GitStatus){
```

Comment:
The code style for this project uses a single space after keywords like `if`, `foreach`, etc and a single space before an opening `{`.